Dockerfile

```diff
@@ -7,8 +7,8 @@ ENV APP_DOWNLOAD_URL https://dl.bintray.com/epam/reportportal/5.0.0-RC-1
 
 ADD ${APP_DOWNLOAD_URL}/service-analyzer_linux_amd64 /service-analyzer
 
-RUN chmod +x /service-analyzer
-
+RUN chmod +x /service-analyzer && \
+    apk add --no-cache ca-certificates
```

Comment:
Please also add these lines to [DockerfileTmpl](https://github.com/reportportal/service-analyzer/blob/master/DockerfileTmpl).

Reply:
Yes, it's done